src/store/dynamodb/mod.rs

```diff
@@ -42,11 +42,10 @@ impl StoreGetAll for DynamoDBStore {
         let res = req.send().await?;
 
         // Build response
-        let products = match res.items {
-            Some(items) => items
-                .into_iter()
-                .map(|v| v.try_into())
-                .collect::<Result<Vec<Product>, Error>>()?,
+        let products: Vec<Product> = match res.items {
+            Some(items) => from_items(items).map_err(|_|
+                // TODO: Find out correct error from underlying error?
+                Error::InternalError("Missing name"))?,
```

Comment:
This already shows a problem: How to map the underlying error to our very specific error `Missing name`. I don't think that the underlying serde errors give us that info: https://github.com/zenlist/serde_dynamo/blob/3.0.0-alpha/src/error.rs

Reply:
Looking into the source code, it looks like it just uses `serde_dynamo::ErrorImpl::Message`, which might contain the right message - but that's not a guarantee. The best thing would be to write a small sample using the Product struct and `serde_dynamo`.